### ISSUE TITLE
Representation of status within the referral form

### ISSUE THREAD

**Issue by hmakandi** [OpenSRP/opensrp-client-chw#1252]
We have a situation that needs to look at 2 ways of representing status within the CHW app under referrals module. We need to show that:

1. The followup is due(blue) or overdue(red) as per the normal convention in OpenSRP
2. In addition to also show that the client has visited the health facility or ADDO(This was in green)

This should allow the CHW to know which of the follow up visits are due/overdue, and which of the clients have gone to the HF/ADDO.

Currently, we have tried the 3 below and they are not aesthetically pleasing.
![image (5)](https://user-images.githubusercontent.com/18394882/86380687-0ea40280-bc95-11ea-971e-a57c132fd03a.png)
![image (4)](https://user-images.githubusercontent.com/18394882/86380703-1368b680-bc95-11ea-884c-b2d018e47e71.png)
![image (3)](https://user-images.githubusercontent.com/18394882/86380713-1794d400-bc95-11ea-9085-50deba0fad79.png)

**Comment by rowo** [comment]
@hmakandi I don't think we can use color in the button. See below: I'm leaning towards a green checkmark and text to indicate someone went on the visit next to the visit that is supposed to be done. We can also go with a checkmark somewhere on the button itself, though it's not super clear why it's there (though I think it it's clearer than the green colors from before). 

Not sure if @mberg has any thoughts on this also. This use case is pretty different from other OpenSRP workflows since it's sort of a half-form completion that says something was done, but data isn't fully captured and it doesn't impact tasking. I don't have enough context to think of other solutions to this, but I know timing is not on design's side.

<img width="770" alt="Screen Shot 2020-07-07 at 1 22 03 AM" src="https://user-images.githubusercontent.com/1584163/86717585-4e9a1b00-bff0-11ea-8c30-3c6205328211.png">
